### ISSUE TITLE
import pipelines not unchecking imported studies

### ISSUE THREAD

**Issue by sheridancbio** [knowledgesystems/pipelines-scrum#850]
## Done Condition (What do we need? Why do we need it? Keep this is small as possible!)
unchecking after normal imports are happening automatically

## Technical Description (How are we going to achieve the above)
This issue was first reported by Tim (in the msk-automation-portal columns) but it was also noticed for genie imports

**Comment by sheridancbio** [comment]
In the importer logs, we were getting messages such as:
2022-03-23 15:00:06 [main] WARN  org.mskcc.cbio.importer.config.internal.GDataImpl - Google sheet 'cancer_studies' contains 1 column headers without printable characters. This will make the values of such columns inaccessible and may cause other processing errors.

From this, I confirmed that the first column header in the cancer_studies sheet was empty. This was replaced with CANCER_STUDIES
